_pages/dev/ssr/server-side-rendering-in-spartacus.md

```diff
@@ -29,288 +42,291 @@ For most situations and setups, is is best to add SSR support to your Spartacus
 
 1. Add the following dependencies to `package.json`:
 
-    ```json
-    "@angular/platform-server": "~10.1.0",
-    "@nguniversal/express-engine": "^10.1.0",
-    "@spartacus/setup": "^3.0.0-rc.2",
-    "express": "^4.15.2"
-    ```
+   ```json
```

Comment:
I wonder if we still need to keep all of this manual information up to date, since schematics takes care of it and it might not be up to date?

Reply:
Not up-to-date for sure.
It's a good idea to have the manual step if the schematics fail for whatever reason.
Adding a note about the outdated deps might be a way to go, as we can't update these every release.